System/Console/ANSI/Codes.hs

```diff
@@ -0,0 +1,161 @@
+-- | Functions that return 'String' values containing codes in accordance with:
+-- (1) standard ECMA-48 Control Functions for Coded Character Sets (5th edition,
+-- 1991); or (2) in the case of 'setTitleCode', the XTerm control sequence.
+--
+-- The reference used for the codes in this module was
+-- <http://en.wikipedia.org/wiki/ANSI_escape_sequences>.
+--
+-- If module "System.Console.ANSI" is also imported, this module is intended to
+-- be imported qualified, to avoid name clashes with functions which return \"\"
```

Comment:
This has now been merged as a separate commit. Could you rebase on top of master and get rid of this?